Test/Unit/Model/RunnerTest.php

```diff
@@ -93,7 +114,46 @@ public function testRun()
             'SomethingDigitalUpgradeHelper\Module\Controller\Adminhtml\Promo\Catalog\Save'
         );
 
+        // Preference
```

Comment:
This isn't a preference. Can you add a comment to describe what we're actually testing here (override of same file in 2 themes).

Reply:
This is updated with a new commit